docs/src/main/paradox/bootstrap/kubernetes.md

```diff
@@ -0,0 +1,149 @@
+# Kubernetes
+
+As of Akka 2.5.15 and Akka-Management 0.18 the recommended way to run Akka Cluster in Kubernetes is to:
+
+* Use Akka Bootstrap with `akka-dns` with cluster formation via DNS SRV records
+* Use a headless service for internal for Akka management/bootstrap so that readiness probes for prod traffic don't interfere with bootstrap 
+* If required use a separate service and/or ingress for user facing endpoints e.g. HTTP gRPC
+
+# Kubernetes Services
+
+For Akka Cluster / Management use a headless service. This allows the solution to not be coupled to k8s as well
+as there is no use case for load balancing across management/remoting ports.
+Set endpoints to be published before readiness checks pass as these endpoints are required to bootstrap the Cluster
+and make the application ready. 
+
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    appName: "akka-cluster-kubernetes"
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
+  name: "akka-cluster-kubernetes"
+spec:
+  ports:
+    - name: management
+      port: 8558
+      protocol: TCP
+      targetPort: 8558
+    - name: remoting
+      port: 2552
+      protocol: TCP
+      targetPort: 2552
+  selector:
+    appName: "akka-cluster-kubernetes"
+  clusterIP: None
+  publishNotReadyAddresses: true
```

Comment:
@TimMoore this is the property for publishing DNS entries for not ready

Reply:
I see, nice! Where's that in docs?

Reply:
Thought that's an url but isn't hm http://service.alpha.kubernetes.io/tolerate-unready-endpoints

Reply:
Only in the swagger for the api server from what I can see. 

I found it via its github issue.